tools/src/main/java/org/apache/rocketmq/tools/command/auth/ListAclSubCommand.java

```diff
@@ -63,7 +63,7 @@ public Options buildCommandlineOptions(Options options) {
         opt.setRequired(false);
         options.addOption(opt);
 
-        opt = new Option("r", "resource", true, "the resource of acl to filter.");
+        opt = new Option("r", "resource", true, "the resources of acl to filter.");
```

Comment:
From the code point of view, this is a single resource, so it should not be changed directly like this.